### ISSUE TITLE
Cannot use mae as loss in regression task

### ISSUE THREAD

**Issue by Promisery** [sberbank-ai-lab/LightAutoML#52]
I'm using LightAutoML 0.2.16. When I define a new task with
`task = Task('reg', loss='mae')`
I get
`sklearn doesn't support in general case mae and will not be used.`
Btw set metric='mae' works fine for me

**Comment by alexmryzhkov** [comment]
Hi @Promisery,

That's pretty normal story - it is a warning that means "for sklearn algos we can't use mae as loss, but for other algos it works fine". In case we can't use the loss for some algos they use default loss for training, others use the loss you set.

Alex
